program/Cargo.toml

```diff
@@ -68,9 +68,10 @@ tensor-vipers = { version = "1.0.1" }
 tensorswap = { version = "0.1.0", package = "escrow-program", git = "ssh://git@github.com/tensor-foundation/escrow.git", features = [
     "cpi",
 ] }
-whitelist-program = { rev = "71e0013", package = "whitelist-program", git = "ssh://git@github.com/tensor-foundation/whitelist.git", features = [
+whitelist-program = { package = "whitelist-program", git = "ssh://git@github.com/tensor-foundation/whitelist.git", branch = "main", features = [
```

Comment:
Can we pin this? So it's obvious what the latest version of this program is built with retroactively.